src/widgets/allergies/allergies-overview.component.tsx

```diff
@@ -28,8 +29,11 @@ export default function AllergiesOverview(props: AllergiesOverviewProps) {
   const [allergiesExpanded, setAllergiesExpanded] = useState<boolean>(false);
   const [isLoadingPatient, patient, patientUuid] = useCurrentPatient();
   const chartBasePath = useChartBasePath();
-  const allergiesPath = chartBasePath + "/" + props.basePath;
   const { t } = useTranslation();
+  const {
+    props: { basePath }
+  } = props;
+  const allergiesPath = chartBasePath + "/" + basePath;
```

Comment:
Widgets are receiving a prop called `props`??? That seems weird and bad. Perhaps I messed something up in https://github.com/openmrs/openmrs-esm-patient-chart/blob/master/src/view-components/widget/widget.component.tsx . The `widgetConfig.props` should be passed through as the actual widget props.